pkg/search/postgres/aggr_func.go

```diff
@@ -0,0 +1,28 @@
+package postgres
+
+import "github.com/stackrox/rox/pkg/postgres/walker"
+
+// AggrFunc is the internal enum representation of the SQL aggregate functions.
+type AggrFunc string
+
+// Defines all the SQL aggregate functions.
+const (
+	Unset AggrFunc = ""
```

Comment:
nit:  IMO for readability, I think it would make sense to name these things like `UnsetAggr`, `CountAggr`, etc.  Particularly for the Unset one as that does not correspond to an aggregation.  I think that woulds make it clear when looking through the calls to `selectQueryField`.  I know doing so is sort of redundant.

Reply:
that is a fair point